app/controllers/api/v1/sessions_controller.rb

```diff
@@ -17,7 +17,7 @@ def render_create_success
       def set_player_id
         return if current_user.player_id.present?
 
-        NotificationService.new(current_user, nil).set_id
+        NotificationService.new(current_user, nil, nil).set_id
```

Comment:
I would do this in a different way. Why don't you use optional parameters?. 
You could update the `initialize(user, matching_users, text)` method in notification service to look like `initialize(user, matching_users = nil, text = nil)` or `initialize(user, _matching_users, _text)` if sometimes those arguments aren't needed, so you don't have to pass `nil` params when you don't need them. Take a look into Rails guides to read more about this :)